src/main/webapp/app/entities/therapeutic-regime/therapeutic-regime-update.component.html

```diff
@@ -83,27 +88,31 @@ <h5 class="list-text-color-bold col-lg-12 mt-4 mb-4"  jhiTranslate="supportiveca
                     <textarea type="text" rows="3"  class="form-control" name="notice" id="field_notice" data-cy="notice"
                               formControlName="notice"></textarea>
                 </div>
-                <div class="form-group col-lg-12">
-                    <label jhiTranslate="supportivecareApp.therapeuticRegime.drug" for="field_drugs">Drug</label>
-                    <select class="form-control" id="field_drugs" data-cy="drug" multiple name="drugs" formControlName="drugs">
-                        <option [ngValue]="getSelected(drugOption, editForm.get('drugs')!.value)" *ngFor="let drugOption of drugs; trackBy: trackById">{{ drugOption.name }}</option>
-                    </select>
-                </div>
-                <div *ngIf="editForm.get(['drugs'])!.invalid && (editForm.get(['drugs'])!.dirty || editForm.get(['drugs'])!.touched)">
-                    <small class="form-text text-danger"
-                           *ngIf="editForm.get(['drugs'])?.errors?.required" jhiTranslate="entity.validation.required">
-                        This field is required.
-                    </small>
-                </div>
-                <div *ngIf="editForm.get(['treatment'])!.invalid && (editForm.get(['treatment'])!.dirty || editForm.get(['treatment'])!.touched)">
-                    <small class="form-text text-danger"
-                           *ngIf="editForm.get(['treatment'])?.errors?.required" jhiTranslate="entity.validation.required">
-                        This field is required.
-                    </small>
+
+                <div class="form-group col-lg-12 ">
+                    <label class="form-control-label" jhiTranslate="supportivecareApp.therapeuticRegime.drug" for="field_drugs" >Drugs</label>
+
+                    <ng-multiselect-dropdown style="display:flex;"
+                        [placeholder]="'...'"
+                        [data]="getSelect2Options(drugs)"
+                        formControlName="drugs"
+                        id="field_drugs"
+                        data-cy="drugs"
```

Comment:
This data-cy changed from `[data-cy="drug"]` to `[data-cy="drugs"]`, does this impact cypress tests or was this purposeful?

Reply:
This was changed by mistake when assigning the correct name for the attributes, should be data-cy=drug, will update it tomorrow

Reply:
corrected